src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/HealthCertificate/CertificateNotificationView.swift

```diff
@@ -0,0 +1,79 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class CertificateNotificationView: UIView {
```

Comment:
I am thinking if the name could be confuse because one could think this view is about the Notification. But here, it's more about the Badge?